config/provisioning/default.sh

```diff
@@ -93,9 +93,11 @@ function provisioning_get_nodes() {
             if [[ ${AUTO_UPDATE,,} != "false" ]]; then
                 printf "Updating node: %s...\n" "${repo}"
                 ( cd "$path" && git pull )
-                if [[ -e $requirements ]]; then
-                    micromamba -n comfyui run ${PIP_INSTALL} -r "$requirements"
-                fi
+            fi
+
+            printf "Ensuring node requirements: %s...\n" "${repo}"
```

Comment:
I think my main concern is that this is kind of noisy output in cases when most are already installed? Maybe there's a more elegant way to check if any need update before output? Feedback welcome of course.

Reply:
(umm, including the lines below too, that suffers the same ills I'd wager)